.travis.yml

```diff
@@ -15,6 +15,11 @@ before_install:
   # Create and start emulator for the script. Meant to race the install task.
   - echo no | avdmanager create avd --force -n test -k "system-images;android-16;default;armeabi-v7a"
   - $ANDROID_HOME/emulator/emulator -avd test -no-audio -no-window &
```

Comment:
I think you want to start the emulator _after_ downloading it.

Reply:
I can't even copy paste right..